tests/integration/cwd-priv.bats

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bats
+
+load helpers
+
+# This test must be particular with how it's run. The user that runs it must have the privileges
+# to chown a directory it creates away from itself, but not have CAP_DAC_OVERRIDE to override
+# the fact it was chowned away. However, it must also be able to clean up.
+# Thus, this test must be skipped if the UID running the test is root, or the user doesn't have sudo privileges.
+function setup() {
+	if [ "$EUID" -eq 0 ]; then
+		skip "This test must be run as a non-root user"
+	fi
+	if ! sudo -v; then
+		skip "This test must be run as a user with sudo privileges"
+	fi
+
+	# Specifying user in a spec can't work in rootless containers that don't have idmap.
+	[[ "$ROOTLESS" -ne 0 ]] && requires rootless_idmap
+
+	teardown_busybox
+	setup_busybox
+
+	SOURCE=$(mktemp -d "$BATS_TMPDIR/cwd.XXXXXX")
+}
+
+function teardown() {
+	sudo rm -rf "$SOURCE"
+	teardown_busybox
+}
+
+# Verify a cwd owned by the container user can be chdir'd to,
+# even if runc doesn't have the privilege to do so.
+@test "runc create sets up user before chdir to cwd" {
+	USER=10000001
+	sudo chown $USER:$USER "$SOURCE"
+	ls -ld "$SOURCE" >&3
+	update_config " .mounts += [{\"source\": \"$SOURCE\", \"destination\": \"$SOURCE\", \"options\": [\"bind\"]}]
```

Comment:
if you can use single quotes around the whole jq statement, you won't have to escape the double quotes. Shell env vars can still be used, e.g.

```suggestion
	update_config ' .mounts += [{"source": "'"$SOURCE"'", "destination": "'"$SOURCE"'", "options": ["bind"]}]
...